src/main/java/com/asn/counter/controller/CounterController.java

```diff
@@ -0,0 +1,45 @@
+package com.asn.counter.controller;
+
+import com.asn.counter.service.CounterValueService;
+import com.asn.counter.bo.Counter;
+import com.asn.counter.repository.CounterRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/counters")
+@RequiredArgsConstructor
+public class CounterController {
+
+    private final CounterRepository counterRepository;
+    private final CounterValueService counterValueService;
+
+    @GetMapping()
+    public List<Counter> getAllCounters(){
+        return counterRepository.findAll();
+    }
+
+    @PatchMapping("/{id}")
```

Comment:
Where is REST? https://restfulapi.net/

Reply:
Patch is a partial update. You do here `increment`, so it doesn't look like REST

Reply:
Read about REST resources archetypes: https://cagline.medium.com/restful-web-services-ddafb8019f2e